src/models/user.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const userSchema = Schema({
+  userName: {
```

Comment:
"username" is a single word; you can make it all lowercase.

Reply:
I changed it. Thanks for the feedback.